godel_scan_analysis/src/keyence_scan_server.cpp

```diff
@@ -0,0 +1,79 @@
+#include "godel_scan_analysis/keyence_scan_server.h"
+
+#include <pcl_ros/transforms.h>
+#include <pcl/filters/voxel_grid.h>
+
+/*
+*/
+
+godel_scan_analysis::ScanServer::ScanServer(const std::string& world_frame, 
+                                            const std::string& scan_frame)
+  : map_(new ColorCloud)
```

Comment:
Aren't there any risk to omitting the parenthesis when calling a class constructor?